sonar-text-plugin/src/main/resources/org/sonar/plugins/secrets/configuration/google-recaptcha.yaml

```diff
@@ -27,6 +27,14 @@ provider:
             # ReCaptcha official testing secret key
             - "6LeIxAcTAAAAAGG-vFI1TnRWxMZNFuojJ4WifJWe"
       examples:
+        - text: |
+            # noncompliant example
+            props.set("recaptcha_secret", "6LcaQa4mAAAAAFvhmzAd2hErGBSt4FC-BPzm4cNS")
+          containsSecret: true 
+          match: 6LcaQa4mAAAAAFvhmzAd2hErGBSt4FC-BPzm4cNS
+        - text: |
+            # compliant example
+            props.set("recaptcha_secret", System.getenv("RECAPTCHA_SECRET"))
```

Comment:
It does not compile and I think it's because it's missing the match an containsSecret fields here.

Always try compiling before assigning for review ! 😄

Reply:
Ohh, this is interesting.
I actually ran the test GoogleUniqueNameCheckTest and it was successful.
But when I run `mvn test` it also fails.

Reply:
Strange, I hope there's no issue with the tests 😅